test/algorithms/pes_samplers/test_bopes_sampler.py

```diff
@@ -155,6 +158,41 @@ def test_vqe_bootstrap(self):
         np.testing.assert_almost_equal(result.points, ref_points)
         np.testing.assert_almost_equal(result.energies, ref_energies)
 
+    @requires_extra_library
```

Comment:
We just merged a new approach for dealing with optional dependencies after Terra introduced lazy loading. Can you please update the test accordingly? You should find plenty of migration examples in the PR #535